### DISCUSSION TITLE
How does the super seeding mode react to a fake seed?

### DISCUSSION THREAD

**Discussion by Heinz14** [arvidn/libtorrent#7801] [Q&A] [answered]
How does the super seeding mode react to a fake seed?
Does it check seeds if a peer doesn't get data? Or can a fake seed block seeding / reseeding because super seeding mode blindly believes every seed to be real?

**Comment by arvidn** [comment, marked as answer]
super seeding is a way to flip over who decides which pieces to transfer. Normally it's the downloader. A super-seeder instead doesn't advertise itself as a seed, but only as having the one or two pieces it wants the peer to request.

A peer that advertises itself to be a seed will be disconnected (since there's no point for seeds to stay connected to each other).

I can't think of a scenario where a fake seed would be a problem.
